client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/index.tsx

```diff
@@ -60,6 +61,9 @@ const designSetup: Step = function DesignSetup( { navigation, flow } ) {
 	const siteVerticalId = useSelect(
 		( select ) => ( site && select( SITE_STORE ).getSiteVerticalId( site.ID ) ) || undefined
 	);
+	const { data: siteVerticalImages = [], isLoading: isLoadingSiteVerticalImages } =
+		useVerticalImagesQuery( siteVerticalId || '', { limit: 1 } );
```

Comment:
I think we have to keep `include_parents`?

Reply:
Oh I've added that as default option values [here](https://github.com/Automattic/wp-calypso/pull/63447/files#diff-d4c9157fd0b4d3b57c62bcdec8d94deb201c57b6318a1737a501839d4d9aa338R8).

Reply:
Okay, I don't notice that 👍